### ISSUE TITLE
Problem with bootloader!

### ISSUE THREAD

**Issue by heathkimdev** [theracermaster/Gigabyte-GA-Z77X-DSDT-Patch#39]
I'm satisfied all functions are fully functional!

but there's a problem with bootloader doesn't list up any NTFS Volumes at devices should be listed.

Then, I tried to replace it to multibeast's one.
After multibeast's clover bootloader installation completed, I rebooted,

Not changed at all, I checked install log.
It seems properly installed according to log and file access time.

But no NTFS drives appears.

I'm asking this problem because I also installed windows, I mean use dual boot system. 

Thanks!

**Comment by theracermaster** [comment]
Did you install Windows in UEFI mode? You shouldn't need to boot from a NTFS partition. If you really need to, try setting `Legacy` to true in config.plist/Boot/Scan, and add a NTFS EFI driver.

**Comment by heathkimdev** [comment]
Thanks, It list up NTFS devices but can't boot to windows.

when I select window device and press enter, it says
`Reboot and Select proper Boot device or Insert Boot Media in selected Boot device and press a key`

I added `NTFS.efi` from https://github.com/JrCs/CloverGrowerPro/blob/master/Files/NTFS/x64/NTFS.efi to `/EFI/CLOVER/drivers64uefi/`

If that `ntfs.efi` is the problem, where can I download official `ntfs.efi`?

I installed windows and sierra in UEFI mode, and I could boot them with multibeast's bootloader.

What does 'You shouldn't need to boot from a NTFS partition' mean?
Is there way to boot windows not to boot to NTFS partition?

Currently, I don't have multibeast's clover backup and multibeast's bootloader install doesn't work at all.

Thanks in advance!
